server/db/models/review.js

```diff
@@ -0,0 +1,25 @@
+var mongoose = require('mongoose');
+
+
+var ReviewSchema = new mongoose.Schema({
+  stars: {
+    type: Number,
+    min: 1,
+    max: 5
+  },
+  description: {
```

Comment:
if description is the review itself, maybe would be more semantic to name it "content" or something like that